src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx

```diff
@@ -3,34 +3,35 @@ title: AWS Lambda Extensions Telemetry API integration
 tags:
   - Integrations
   - Amazon integrations
-  - AWS integrations list
+  - Lambda Extension
 ---
+**Work in progress**
 
-The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch. Send your logs, events, metrics and traces by adding the extension as the layer, and setting up your New Relic license key as an environment variable.
+The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch. Send your logs, events, metrics and traces by adding the extension as the layer, and get insights from your data instently on New Relic One platform.
 
 ## Installation
 
-
 1. In the AWS Console go to **Lambda** > **Functions** and choose the function you'd like to ingest data from.
 2. Select **Layers** > **Add Layer**.
 3. In **Choose a layer** select the **Specify a layer by providing the ARN** option.
-4. Navigate to [https://layers.newrelic-external.com/](https://layers.newrelic-external.com/), search for AwsLambdaExtension in your region to find the arn with the most recent layer version, and then paste that in the Arn field back in your AWS console. 
-6. Go to **Configuration**, select **Environment variables** and add **Key: LICENSE_KEY  Value: <var>LICENSE_KEY</var>**. 
+4. **To Be Done** Navigate to [https://layers.newrelic-external.com/](https://layers.newrelic-external.com/), search for AwsLambdaExtension in your region to find the arn with the most recent layer version, and then paste that in the Arn field back in your AWS console. 
```

Comment:
@4demos What's the **To Be Done** referring to? Should this be removed?

Reply:
actually now the extension is still not awilable on https://layers.newrelic-external.com/. We anticipate it to be there soon but it's not there right now.  So, now it's 4a.